### PR TITLE
Srahulbadami patch 1

### PR DESCRIPTION
**Pull request by srahulbadami**
Thank you for your pull request!

Please acknowledge each of the following before submitting your pull request.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- leave it unchecked. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by robbyoconnor** [comment]
You're not following directions. Actually read the text there.

**Comment by robbyoconnor** [comment]
Your image is NOT 240x240.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Image is not 240 x 240.That number isn't there for show. It's a requirement.

**Comment by robbyoconnor** [comment]
I am closing this. Please make your PR using the `gh-pages` branch in your fork. Next time, read the text in the pull request template.

**Comment by srahulbadami** [comment]
what should i do for a live link or screenshot ??

**Comment by abishekvashok** [comment]
@srahulbadami
For a live link go the url:
`https://username.github.io/repositoryName` 
and on the repository settings configure the branch to be deployed to be your current branch.

So your live link for this repo will be :
https://srahulbadami.github.io/gci16.fossasia.org
If you turned the  branch to be deployed to be the branch from which you are making the PR from

**Comment by robbyoconnor** [comment]
Please open a new pull request...you have a syntax error in your YAML.

File: _data/students.yml [hunk 1]
```diff
@@ -33,6 +33,13 @@
   bio: 17 Years Old, Future Silicon Valley Resident, Loves Music and Creating!
   lat: 30.011387
   lng: 31.298684
+  
+- name: S Rahul Badami
+  github: srahulbadami
+  image: badami.jpg
+  twitter: srahulbadami
+  email:srahulbadami@gmail.com
```
Comment (robbyoconnor) [inline _data/students.yml:41]: @srahulbadami you need a space here.

**Comment by abishekvashok** [comment]
Also please see the PR body, read it and make the changes like to replace the text which asks you to replace it with the ones explaining what you have done in this PR and adding your live link to it or a screenshot

**Comment by srahulbadami** [comment]
There is no option to deploy the branch

**Comment by srahulbadami** [comment]
@Abhi2424shek: There is no option to deploy the branch

**Comment by abishekvashok** [comment]
Please see the screenshot:
screenshot_2016-12-28-17-38-14-984_com android chrome

**Comment by abishekvashok** [comment]
See the github pages section pls

**Pull request closed without merging**